README.md

```diff
@@ -96,7 +96,8 @@ app.post(
   express.json(),
   // Decrypt the submission and attachments
   async function (req, res, next) {
-    const submission = formsg.crypto.decryptWithAttachments(
+    // Note that this function is only available from version 0.9.0 onwards
```

Comment:
I think this line is not necessary, as it's already documented below. I'd also like this section to be a "quick start" that is as short, friendly and un-intimidating as possible. Do you think we can simplify further, say by combining `/submissions` with `/submissions-attachment`?

Reply:
addressed in https://github.com/opengovsg/formsg-javascript-sdk/pull/72/commits/3c49a9c3e1c1e88752862e5f94e90109be517fe3, could you help take a look to see if it makes sense please?